lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -1,30 +1,53 @@
 
+<div class="text-section">
+    <h3 id="program-<%- id %>" class="title"><%- gettext(name) %></h3>
+    <div class="meta-info">
+        <div class="organization"><%- orgList %></div>
+        <div class="category">
+            <span class="category-text"><%- gettext(type) %></span>
+            <span class="xseries-icon" aria-hidden="true"></span>
+        </div>
+    </div>
+  <% if (progress) { %>  
+    <p class="certificate-status"><%= interpolate(
+        gettext('You have earned Verified Certificates in %(completed_courses)s of the %(total_courses)s courses so far.'),
```

Comment:
The presence of a verified certificates doesn't necessarily indicate course code completion. It's possible to define any certificate type as suitable for completion of a course code.

Could you make this more general? For example, "You have completed %(completed_courses)s of %(total_courses)s courses in this program."